File: Course_01/C++ Language/EstruturaRepetitiva/fatorial.cpp
#include <iostream>

using namespace std;

int main()
{
    int n, i, fatorial(1);

    cout << "Digite o valor de N: ";
    cin >> n;

    for (i = 1; i <= n; i++) {
        fatorial *= i;
    }

    cout << "FATORIAL = " << fatorial << endl;
    return 0;
}

File: Course_01/Java Language/Estrutura repetitiva/fatorial.java
package Repetitivo;

import java.util.Scanner;

public class fatorial {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n, i, fatorial;
		
		System.out.print("Digite o valor de N: ");
		n = sc.nextInt();
		
		fatorial = 1;
		
		for (i = 1; i <= n; i++) {
			fatorial *= i; 
		}
		
		System.out.println("FATORIAL = " + fatorial);
		
		sc.close();
	}

}
